frame/contracts/src/storage/meter.rs

```diff
@@ -224,6 +224,18 @@ impl Diff {
 	}
 }
 
+/// The beneficiary of a contract termination.
+type BeneficiaryOf<T> = AccountIdOf<T>;
+
+/// The status of a contract.
+///
+/// In case of termination the beneficiary is indicated.
+#[derive(RuntimeDebugNoBound, Clone, PartialEq, Eq)]
+pub enum ContractStatus<T: Config> {
+	Alive,
+	Terminated(BeneficiaryOf<T>),
```

Comment:
Please use a struct like variant in order to add a name to a value. Type aliases add mental load by creating another indirection.